deploy-manage/production-guidance/optimize-performance/approximate-knn-search.md

```diff
@@ -122,13 +123,11 @@ You can check the current value in `KiB` using `lsblk -o NAME,RA,MOUNTPOINT,TYPE
 ::::
 
 
-## Use Direct IO when the vector data does not fit in RAM
+## Use on-disk rescoring when the vector data does not fit in RAM
```

Comment:
is the old direct IO guidance still valid in 9.1? did your team plan to not move forward with it (i.e. it is going from preview to removed)? Wonder if we need to keep it visible, but if it is going from preview to removed, your approach of just editing it out is ok.